src/Workspaces/Core/Desktop/Workspace/FileTextLoader.cs

```diff
@@ -11,11 +11,17 @@
 
 namespace Microsoft.CodeAnalysis
 {
+    [System.Diagnostics.DebuggerDisplay("{GetDebuggerDisplay}")]
```

Comment:
Every other occurrence in https://github.com/dotnet/roslyn/search?utf8=%E2%9C%93&q=GetDebuggerDisplay has the `()` for the method call.  Did you test that this actually works?